translation/en/RedirectedResources/assets/abdata/h/list/00_00/personality_voice_c00_03_00/translation.txt

```diff
@@ -1052,10 +1052,10 @@
 //ゴムがこすれて……良かったわ=
 //いつかゴムを破っちゃいそうな勢いよね=
 //ゴムにはゴムなりの良さがあるのよね=
-//まだちょっと出てる……=
+//まだちょっと出てる……=Wait~, don't pull it out yet
 //しばらく抜かないでいて=
 //君のおいしいザーメン、下のお口でいただいちゃいました♪=
-//この余韻がまたいいの……=
+//この余韻がまたいいの……=This sensation again~...
```

Comment:
This is a little difficult since in a literal sense, she's talking about being in an afterglow state again. Not sure how to word it correctly without making it sound awkward.

Reply:
What do you think of: "This delighting/delightment/delight again feels great..." Not sure which one of the three words is more appropriate though.

Reply:
It might be better to remove the word "again" and casually hint that this isn't the first time instead.
"I'm loving this afterglow feeling..."